core/src/main/java/com/datastrato/gravitino/connector/BaseCatalog.java

```diff
@@ -57,6 +65,13 @@ public abstract class BaseCatalog<T extends BaseCatalog>
   // The object you used is not stable, don't use it unless you know what you are doing.
   @VisibleForTesting public static final String CATALOG_OPERATION_IMPL = "ops-impl";
 
+  // Different catalogs may have different authorization implementations, this variable is used as a
+  // key in properties of catalogs to inject custom authorization to Gravitino.
+  public static final String AUTHORIZATION_IMPL = "authorization-impl";
```

Comment:
Can we rename the property to `authorization-provider`? Also I would suggest to define the property name `Catalog` for users to better understand this property name. Besides, you need to add the property  to `BaseCatalogPropertiesMetadata`.

Also don't forget to add to the doc.

Reply:
DONE.